app/components/filter-component.js

```diff
@@ -3,13 +3,16 @@ import Component from '@ember/component'
 export default Component.extend({
   init() {
     this._super(...arguments)
+    this.set('filterCategories', this.get(`filterCategories.${this.name}`))
```

Comment:
Why?

Reply:
You should not overwrite properties (what if `filterCategories` changes from outside the component?)..

Reply:
With the redefinition of filter into an computed property. I couldn't directly hand over the filters of a category, only the entire Object.